### DISCUSSION TITLE
72

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#730] [Comments]
http://jojoldu.tistory.com/72

**Comment by jojoldu** [comment]
작성자: ㅇㅇㅅ

잘보고갑니다. 덕분에 공부했습니다.

**Comment by Kyeongrok** [comment]
유용한 정보에 감탄을 금치 못하고 갑니다.
소스코드를 예쁘게 잘 올리셨네요.
소스코드는 어떤 툴을 이용해 올리시나요?

**Comment by jojoldu** [comment]
안녕하세요? 칭찬감사합니다!
소스코드는 블로그에 highlight.js와 markdown.css를 올린뒤 자체 제작한 [markdown 컨버터](https://github.com/jojoldu/markdown-tistory)를 사용해서 올렸습니다.

**Comment by whdms705** [comment]
너무나 좋은 글이었습니다. 
aop에 대해 좀더 이해할 수 있게 많은 도움이 되었습니다!
감사합니다.

**Comment by jojoldu** [comment]
@whdms705 넵 감사합니다 :)

**Comment by namhokim** [comment]
도움이 많이 되었습니다.

**Comment by namhokim** [comment]
@AfterReturning("updateUser(user)") 에 argNames 인자를 주면 더 좋을 것 같습니다.
예)
@AfterReturning(value = "updateUser(user)", argNames = "user")

**Comment by namhokim** [comment]
인자가 하나일 때는 없는 게 낫겠군요.

**Comment by youngjinmo** [comment]
<토비의 스프링> 스터디를 진행하며 AOP 파트에서 막혀서 힘들었는데, 올려주신 내용과 코드덕분에 한 줄, 한 줄 따라치면서 이해할 수 있었습니다. 정말 감사드립니다!
